src/gitops/components/shared/ApplicationList.tsx

```
@@ -111,8 +111,7 @@ const ApplicationList: React.FC<ApplicationProps> = ({
     [listAllNamespaces, namespace],
   );
 
-  const { searchParams, sortBy, direction, getSortParams } =
-    useGitOpsDataViewSort(columnSortConfig);
+  const { searchParams, sortBy, direction, getSortParams } = useGitOpsDataViewSort(columnSortConfig);
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)